tests/src/test_lambda_alerting.py

```diff
@@ -63,6 +63,43 @@ def event_dyn_props_init(aws_props_init):
     return (account_id, region, time_str, epoch_time, version_str)
 
 
+################################################################################################################################
+
+# mocking AWS calls done in lambda_alerting
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
I would think on the scope, probably you can use a broader scope like "module" or "session" to improve efficiency.
Now scope "function" means mocks will be executed for each test function.

Reply:
Agree. Changed to module scope.
Had to switch to unittest.mock functionality, as pytest wrapper doesn't allow anything but "function" scope